modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/dto/v1_0/converter/ObjectValidationRuleDTOConverter.java

```diff
@@ -129,10 +130,18 @@ private ObjectValidationRuleSetting _toObjectValidationRuleSetting(
 					});
 				setValue(
 					() -> {
-						if (!objectValidationRuleSetting.compareName(
+						if (FeatureFlagManagerUtil.isEnabled("LPS-187854") &&
+							!(objectValidationRuleSetting.compareName(
 								ObjectValidationRuleSettingConstants.
-									NAME_KEY_OBJECT_FIELD_ID) ||
-							!objectValidationRuleSetting.compareName(
+									NAME_KEY_OBJECT_FIELD_ID) &&
+							  objectValidationRuleSetting.compareName(
+								  ObjectValidationRuleSettingConstants.
+									  NAME_OUTPUT_OBJECT_FIELD_ID))) {
```

Comment:
Do we need to also check NAME_OUTPUT_OBJECT_FIELD_ID here? It seems that it's already covered in the if condition below. We should keep NAME_OUTPUT_OBJECT_FIELD_ID working as before, even without this feature flag.

Reply:
Yes, we need to keep the antique behavior.
After the Feature Flag removal we will have the follow snippet:
```java
if (!(objectValidationRuleSetting.compareName(
	        ObjectValidationRuleSettingConstants.
		        NAME_KEY_OBJECT_FIELD_ID) ||
	objectValidationRuleSetting.compareName(
		ObjectValidationRuleSettingConstants.
			NAME_OUTPUT_OBJECT_FIELD_ID)))
``` 
Indicating that the setValue only accepts those two types of  `objectValidationRuleSetting` names